js/core/clusters/clusters.js

```diff
@@ -53,6 +53,11 @@ define([
   return function(config) {
     var lock = false;
 
+    // getter for config.cluster
+    window.cluster = function() {
```

Comment:
there is only one way to set it, there is only one way to get it globally, when `config` is not is the current scope